.github/workflows/pylint.yml

```diff
@@ -0,0 +1,34 @@
+name: Pylint
```

Comment:
if we added another language, like JS, to the mix here.. what would that look like?

Reply:
we would add either:
a) another workflow yml
b) break this into 2 jobs and run one against all the JS files